libcxx/docs/Hardening.rst

```diff
@@ -458,7 +458,7 @@ Hardened containers status
       - Partial
       - N/A
     * - ``bitset``
-      - ❌
+      - ✅
```

Comment:
I'm still hesitant to list `valarray` as fully hardened given the numerous helper classes -- that would require some investigation!